tests/sentry/monitors/test_monitor_consumer.py

```diff
@@ -81,7 +84,7 @@ def send_message(
                     KafkaPayload(b"fake-key", msgpack.packb(wrapper), []),
                     partition,
                     1,
-                    datetime.now(),
```

Comment:
are these separate values or no? `start_time` would be when the SDK generates the payload and this would be when relay receives it, right?

Reply:
They are slighlty different yes, but I am OK for the tests keeping them the same for now.

This one is actually the timestamp of when the message is placed into the kafka queue.

Reply:
sounds good just wanted to double check